### ISSUE TITLE
Blenderplayer crashes with Python 3.9, invoking _PyObject_GC_New

### ISSUE THREAD

**Issue by mysticfall** [UPBGE/upbge#1341]
After my Manjaro unstable installation upgraded Python to 3.9, I can no longer run UPBGE project as it segfaults immediately with the following trace:
```
#0  0x00007fd728569927 _PyObject_GC_New (libpython3.9.so.1.0 + 0x10e927)
#1  0x00007fd72856bedc PyDict_New (libpython3.9.so.1.0 + 0x110edc)
#2  0x0000564722071c7b main (blenderplayer + 0xf40c7b)
#3  0x00007fd727f97152 __libc_start_main (libc.so.6 + 0x28152)
#4  0x00005647220cacde _start (blenderplayer + 0xf99cde)
```

**Comment by mysticfall** [comment]
It occurred to me that this might have been caused by running a Python 3.8 project without updating its dependencies. I will test it again with an updated project next week and see if it resolves the problem.

**Comment by mysticfall** [comment]
I was able to do more tests last night and unfortunately the problem persisted even after I updated all Python 3.8 references from my project.

I also found that the crash only happens with `blenderplayer` but it runs fine when I run it using the `Embedded Start` feature.

Currently, I'm doing a wild goose hunt as I have no better idea how to fix it. It's probably not really related, but I found 5edba9b42f684bf and 0133bcaf38f6ecb5d from the commit log which mentions Python 3.9 and a crashing problem.

I'll have some free time tomorrow so I will try to dig deeper. But as I don't know C++ or Blender internals, it's unlikely that my search would be fruitful.
